chain/rosetta-rpc/src/utils.rs

```diff
@@ -72,7 +76,7 @@ where
     }
 }
 
-#[derive(Debug, Clone, PartialEq, derive_more::AsRef, derive_more::From)]
+#[derive(Debug, Clone, Eq, derive_more::AsRef, derive_more::From)]
```

Comment:
Won’t derive just work?
```suggestion
#[derive(Debug, Clone, PartialEq, Eq, derive_more::AsRef, derive_more::From)]
```